source/math/FloatVectorList.ooc

```diff
@@ -14,12 +14,17 @@
 * You should have received a copy of the GNU Lesser General Public License
 * along with this software. If not, see <http://www.gnu.org/licenses/>.
 */
-import VectorList
+use ooc-collections
 import math
-FloatVector: class extends VectorList<Float> {
+import IntExtension
+
+FloatVectorList: class extends VectorList<Float> {
```

Comment:
In the future, keep in mind that it's never a good idea to modify a file and rename it in the one and same commit.